website/content/docs/connect/ca/vault.mdx

```diff
@@ -181,124 +171,190 @@ The configuration options are listed below.
 
-# Needed for Consul 1.11+
-path "/sys/mounts/connect_inter/tune" {
-  capabilities = [ "update" ]
-}
+To use Consul-managed PKI paths, ensure no PKI secrets engines are mounted at
+`RootPKIPath` and `IntermediatePKIPath`.
 
-path "/connect_root/*" {
-  capabilities = [ "create", "read", "update", "delete", "list" ]
-}
+Then, attach the following Vault ACL policy to the CA provider's
+[Vault token](#token) or [auth method](#authmethod):
+
+1. Allow Consul to create and manage both PKI engines:
+
+  <CodeBlockConfig filename="consul-managed-pki-policy.hcl">
+
+  ```hcl
+  path "/sys/mounts/<root_pki_path>" {
+    capabilities = [ "create", "read", "update", "delete", "list" ]
+  }
+
+  path "/sys/mounts/<intermediate_pki_path>" {
+    capabilities = [ "create", "read", "update", "delete", "list" ]
+  }
+
+  path "/sys/mounts/<intermediate_pki_path>/tune" {
+    capabilities = [ "update" ]
+  }
+  ```
+  </CodeBlockConfig>
+
+1. Allow Consul full use of both PKI engines:
+
+  <CodeBlockConfig filename="consul-managed-pki-policy.hcl">
+
+  ```hcl
+  path "/<root_pki_path>/*" {
+    capabilities = [ "create", "read", "update", "delete", "list" ]
+  }
+
+  path "/<intermediate_pki_path>/*" {
+    capabilities = [ "create", "read", "update", "delete", "list" ]
+  }
+  ```
+
+  </CodeBlockConfig>
+
+1. Allow Consul to renew its Vault token if the token is renewable.
+   The rule enables the token to be renewed whether it is provided
+   [directly](#token) in the CA provider configuration or presented in an [auth method](#authmethod).
+
+  <CodeBlockConfig filename="consul-managed-pki-policy.hcl">
+
+  ```hcl
+  path "auth/token/renew-self" {
+    capabilities = [ "update" ]
+  }
+
+  path "auth/token/lookup-self" {
+    capabilities = [ "read" ]
+  }
+  ```
+  
+  </CodeBlockConfig>
+
+#### Additional Vault ACL policies for sensitive operations
+
+Additional Vault permissions are required while you perform the
+following CA configuration changes:
+- Changing the provider from Vault to a different provider, such as Consul's built-in provider
+- Changing the `RootPKIPath`
+
+Those configuration modifications trigger a root CA change that requires an
+extremely privileged root cross-sign operation. 
+For that operation to succeed, the CA's [Vault token](#token) or
+[auth method](#authmethod) must contain the following rule:
 
-path "/connect_inter/*" {
-  capabilities = [ "create", "read", "update", "delete", "list" ]
+<CodeBlockConfig filename="temporary-sensitive-operation-pki-policy.hcl">
+
+```hcl
+path "<root_pki_path>/root/sign-self-issued" {
+capabilities = [ "sudo", "update" ]
 }
 ```
-
 </CodeBlockConfig>
 
-<!-- Reference style links -->
-[`ca_config`]: /docs/agent/config/config-files#connect_ca_config
-[`ca_provider`]: /docs/agent/config/config-files#connect_ca_provider
-[`/connect/ca/configuration`]: /api-docs/connect/ca#update-ca-configuration
+We recommend adding this elevated privilege just before the CA provider
```

Comment:
@rboyer : In practice... how would a Vault user actually accomplish this?

If I understand correctly, a Vault token's policies are immutable. That would mean you'd need to:
- Create a new Vault token with the elevated policy
- Apply the new Vault token to the Consul Connect CA configuration
- Perform the operation
- Reapply the previous Vault token with non-elevated permissions to the Consul Connect CA configuration

Alternatively, you could use a Vault auth method, and modify the policy associated with the auth method... But if I understand correctly, that wouldn't actually take effect unless Consul re-logs in using the auth method (after the modification) to get a new token (which contains the modified rules). And I don't know how the user would force Consul to do that... So maybe this is only workable when using a Vault token directly rather than an auth method, even though an auth method is generally preferable for enhanced security?